hack/extract-release.sh

```diff
@@ -15,10 +15,11 @@ cd "${OS_ROOT}"
 
 # Copy the linux release archives release back to the local _output/local/bin/linux/amd64 directory.
 # TODO: support different OS's?
-os::build::detect_local_release_tars "linux-64bit"
+os::build::detect_local_release_tars $(os::build::host_platform_friendly)
```

Comment:
It fails today on any other platform because SERVER doesn't exist.  So same behavior.